boards/silabs/radio_boards/slwrb4180b/slwrb4180b_defconfig

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_ARM_MPU=y
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_GPIO=y
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=38400000
```

Comment:
this should come from dts in Kconfig, it should not need to be set here @gmarull

Reply:
It is indeed set at https://github.com/zephyrproject-rtos/zephyr/blob/main/soc/silabs/silabs_s2/Kconfig.defconfig#L9, so it can be removed here.